### DISCUSSION TITLE
Facing some issues in TailwindPlay.

### DISCUSSION THREAD

**Discussion by ProPhycient5** [tailwindlabs/tailwindcss#10228] [Help] [answered]
I am playing on TailwindPlay. I see that tailwind classnames are not working as expected. 
Below is the link for it 👇🏽, please check let me know.
https://play.tailwindcss.com/YErQLEEU57

**Comment by RobinMalfait** [comment, marked as answer]
Hey! This is because you are using `className` instead of `class`.

**Reply by ProPhycient5** [reply]
Oh, I missed it. Thanks, @RobinMalfait.
